BLSQ/iaso
# IA-1640: Allow multiple OU types selection in the completeness_stats table

When using the new completeness table, selecting multiple OU types (which was doable in the FE) triggered an error. 

## Self proof reading checklist


## How to test

- Go to the completeness table
- Make sure you can filter by OU types (either no selection, one OU type or multiple OU types)
